src/main/scala/no/ndla/articleapi/controller/InternController.scala

```diff
@@ -46,5 +46,10 @@ trait InternController {
     get("/embedurls/:external_subject_id") {
       ArticleContentInformation.getExternalEmbedResources(params("external_subject_id"))
     }
+
+    get("/imagesinarticle/:article_id") {
```

Comment:
Føltes litt voldsomt å ha dette som et "permanent" endepunkt iom. at det bare var ment som en "oneoff" for å kunne vurdere løsning på bilde-mapping. Må vi gjøre det sånn?